dispatcher.py

```diff
@@ -143,10 +143,9 @@ def init_listeners(self):
             self.listeners.append(self.fail_watcher)
         if watch_hang:
             warn_timeout = 60.0 if args.long else 10.0
-            no_output_timeout = float(args.no_output_timeout or 120)
             hang_watcher = listeners.HangWatcher(
                 output_watcher.not_done_worker_ids, self.kill_all_workers,
-                warn_timeout, no_output_timeout)
+                warn_timeout, args.no_output_timeout)
```

Comment:
It seems, it does not break anything, but I would keep the `float` type. At least, it is what HangWatcher expects.

Reply:
Ok, sure, added float().